haechan45th/src/models/userDao.js

```diff
@@ -34,6 +34,20 @@ const getUserId = async (userId) => {
     }
 };
 
+const getUserByEmail = async (email) => {
+    try {
+        const [getUserByEmail] = await dataSource.query(
+            `SELECT users.email FROM users 
+            WHERE users.email = ?`,
+            [email]
+        )
+        return getUserByEmail;
```

Comment:
```suggestion
        const [user] = await dataSource.query(
            `SELECT users.email FROM users 
            WHERE users.email = ?`,
            [email]
        )
        return user;
```
해찬님~ email 뿐만 아니라 password  컬럼과 id 컬럼까지 불러와야 userService에 있는 signIn 메소드의 내부 로직을 수행할 수 있습니다. 현재 상태로서는 로그인 기능에 대해 동작할 수 없으니, 수정 후 e2e 테스트 (postman으로 요청을 보내서 하는)까지 완료해주세요!!

Reply:
![Screenshot 2023-04-26 at 8 25 50 PM](https://user-images.githubusercontent.com/120645757/234561102-de9dfcd9-984d-4ed0-ab4a-cd3785e439eb.png)
옷 넵 빠진게 있었네요! 수정했습니다!